pkgs/by-name/al/altdns/package.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
```

```suggestion
python3.pkgs.buildPythonApplication {
```
It isn't being used.